website/pages/en/index.js

```diff
@@ -29,6 +29,25 @@ class Button extends React.Component {
   }
 }
 
+class Dropdown extends React.Component {
+  render() {
+    const gitpod = template => `https://gitpod.io/#template=${template}/https://github.com/MichalBednarz/gitpod-g8`
```

Comment:
We should probably move this to scalameta

Reply:
Could you create the repo?

Reply:
That's the plan!